SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
 [sdpi_offset=1]
 === Requirements:  From Narratives to Plug-and-Trust Interfaces
 
+For every device or system interface that supports the elements specified in this technical framework, requirements and capabilities are derived from many different standards, types of standards, and standards development organizations (SDO).  Some are technology focused, such as the ISO/IEEE 11073 standards for health device communication, and others are more focused on quality and risk management, such as the ISO/IEC 80001-1 and ISO 14971 standards.  They are all profiled and implemented and tested in a single interface point, and thus need to be integrated in a coherent and cohesive manner.
+
+In order to manage this diversity and wealth of specifications and standards, the following <<hanging_gardens_framework>> graphic was created:
```

Comment:
Generally, I prefer refering to tables, listings and graphics by using Asciidoc references over using "the following".

If we don't use numbering for reference:
`In order to manage this diversity and wealth of specifications and standards, the <<hanging_gardens_framework>> graphic was created.`

Or if we do use numbering for references: 

`In order to manage this diversity and wealth of specifications and standards, the Hanging Gardens Framwork as specified in Figure <<hanging_gardens_framework>> was created.`

Reply:
The `<<hanging_gardens_framework>>` reference does not exist.